timings/state.go

```diff
@@ -44,6 +45,13 @@ type RootTimingsJSON struct {
 	StopTime time.Time `json:"stop-time"`
 }
 
+// TimingsInfo encompasses timings returned by the Get method below.
+// It generally reflects rootTimingJSON, but may hide some of its detials.
```

Comment:
TimingsInfo holds a set of related nested timings and the tags set when they were captured.

Reply:
Ah, I didn't comment on the "detials" typo because my idea was to remove that part, it's a part strange to refer to a non exported typo in a doc comment.